packages/gestalt-datepicker/package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "dependencies": {
     "classnames": "^2.2.6",
-    "react-datepicker": "2.16.0"
+    "react-datepicker": "4.0.0"
```

Comment:
It looks like the current version is `4.11.0`. Can we upgrade all the way to that?

Reply:
They jump from 4 to 11 in like 2 releases. I don't think the left 3 matter based on the comments. I can do that, we won't be any benefit though

Reply:
It looks like there are a lot of bugfixes and other improvements in there: https://github.com/Hacker0x01/react-datepicker/releases

I'm always a bit nervous about `x.0.0` releases, as they're typically followed by a slew of fixes. Unless there's a specific reason not to, we should always try to update to the most current version to get those fixes.

Reply:
DONE

Reply:
4.11.0 Introduces a bug
![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/230123970-a61ac495-50f3-47b6-b87f-48cc42d87ae7.png)

Reply:
No worries. We should upgrade in the near-ish future, but not worth worrying about for now